kalite/topic_tools/__init__.py

```diff
@@ -300,7 +300,8 @@ def get_content_cache(force=False, annotate=False, language=None):
             CONTENT[language] = content
             return CONTENT[language]
         else:
-            CONTENT[language] = softload_json(settings.CONTENT_FILEPATH, logger=logging.debug, raises=False)
+            CONTENT[language] = (softload_sqlite_cache(settings.CONTENT_CACHE_FILEPATH) or
```

Comment:
Oh, my bad. Looks like it's not duplicated in memory, but writes to the values are stored in a [transaction](https://github.com/piskvorky/sqlitedict/blob/master/sqlitedict.py#L441-L450). So they'll only persist across reboots when we call `.commit()` (not `.save`). Only a copy of the value is stored in memory when there's a reference to it somewhere.

Since it uses transactions though, I'm surprised it's working properly, given that python's sqlite3 module is somewhat buggy with transactions.